api/descriptions/policies/vulnerability_serverless_put.md

```diff
@@ -0,0 +1,46 @@
+Updates the vulnerability policy for serverless functions scanned in your pipeline.
+All rules in the policy are updated in a single shot.
+
+This endpoint maps to the policy table in **Defend > Vulnerabilities > Functions** in the Console UI.
+
+
+### cURL Request
+
+The following cURL command overwrites all rules in your current policy with a new policy that has a single rule.
+
+```bash
+$ curl 'https://<CONSOLE>/api/v1/policies/vulnerability/serverless?project=<PROJECT_NAME>' \
+  -X PUT \
+  -u <USER> \
+  -H 'Content-Type: application/json' \
+  -d \
+'{
+  "rules": [
+    {
+      "name": "<RULE_NAME>",
+      "collections": [
+        {
+           "name":"<COLLECTION_NAME>",
+        }
+      ],
+      "alertThreshold": {
+        "value": 1,
+        "disabled": false
+      },
+      "blockThreshold": {
```

Comment:
@KICKdesigns For this type of rule/policy. the 'block' action isn't applicable. Even though the API accepts it without an error, it silently drops the `blockThreshold` stanza. To keep the curl commands tight, please drop this stanza from the example.

You can quickly see which parameters are accepted/not accepted by reviewing the relevant rule dialog in the UI.